### ISSUE TITLE
Cannot unpack array with string keys 

### ISSUE THREAD

**Issue by assada** [DarkGhostHunter/Laraload#3]
```
[2020-01-22 16:45:58] local.ERROR: Cannot unpack array with string keys {"userId":1,"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Cannot unpack array with string keys at /var/www/vendor/darkghosthunter/laraload/src/Http/Middleware/LaraloadMiddleware.php:86)
[stacktrace]
#0 /var/www/vendor/darkghosthunter/laraload/src/Http/Middleware/LaraloadMiddleware.php(58): DarkGhostHunter\\Laraload\\Http\\Middleware\\LaraloadMiddleware->conditionIsTrue()
#1 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(212): DarkGhostHunter\\Laraload\\Http\\Middleware\\LaraloadMiddleware->terminate(Object(Illuminate\\Http\\Request), Object(Illuminate\\Http\\Response))
#2 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(183): Illuminate\\Foundation\\Http\\Kernel->terminateMiddleware(Object(Illuminate\\Http\\Request), Object(Illuminate\\Http\\Response))
#3 /var/www/public/index.php(60): Illuminate\\Foundation\\Http\\Kernel->terminate(Object(Illuminate\\Http\\Request), Object(Illuminate\\Http\\Response))
#4 {main}
"} 
```

**Comment by assada** [comment]
https://github.com/DarkGhostHunter/Laraload/blob/master/src/Http/Middleware/LaraloadMiddleware.php#L88

Temp fix:
```php
return strpos($condition[0], '@')
            ? $this->app->call(... $condition)
            : $this->app->make($condition[0])(...array_values($condition[1] ?? []));
```

**Comment by assada** [comment]
Also got new issue:
```
[2020-01-22 17:20:42] local.ERROR: Call to undefined function DarkGhostHunter\Preloader\opcache_get_status() {"userId":1,"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to undefined function DarkGhostHunter\\Preloader\\opcache_get_status() at /var/www/vendor/darkghosthunter/preloader/src/Opcache.php:29)
[stacktrace]
#0 /var/www/vendor/darkghosthunter/preloader/src/Opcache.php(39): DarkGhostHunter\\Preloader\\Opcache->getStatus()
#1 /var/www/vendor/darkghosthunter/preloader/src/Preloader.php(190): DarkGhostHunter\\Preloader\\Opcache->isEnabled()
#2 /var/www/vendor/darkghosthunter/preloader/src/Preloader.php(152): DarkGhostHunter\\Preloader\\Preloader->canGenerate()
#3 /var/www/vendor/darkghosthunter/laraload/src/Laraload.php(60): DarkGhostHunter\\Preloader\\Preloader->generate()
#4 /var/www/vendor/darkghosthunter/laraload/src/Http/Middleware/LaraloadMiddleware.php(59): DarkGhostHunter\\Laraload\\Laraload->generate()
#5 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(212): DarkGhostHunter\\Laraload\\Http\\Middleware\\LaraloadMiddleware->terminate(Object(Illuminate\\Http\\Request), Object(Illuminate\\Http\\Response))
#6 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(183): Illuminate\\Foundation\\Http\\Kernel->terminateMiddleware(Object(Illuminate\\Http\\Request), Object(Illuminate\\Http\\Response))
#7 /var/www/public/index.php(60): Illuminate\\Foundation\\Http\\Kernel->terminate(Object(Illuminate\\Http\\Request), Object(Illuminate\\Http\\Response))
#8 {main}
```

**Comment by DarkGhostHunter** [comment]
Got it.

**Comment by mfjordvald** [comment]
This got released as 1.0.1 while there is already a 1.1.0 out, so composer is not picking up on this fix. Retag a 1.1.1 release?

**Comment by DarkGhostHunter** [comment]
Crap. I'll have to retag it then.
